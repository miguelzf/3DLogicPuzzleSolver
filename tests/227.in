- - - - - -
- - P - X X
- C - - - -
- - - Y - -
- X - - L -
- X - - - X

- X - - - X
- X B - Y -
- - - - - -
- - G - - C
- - - L X X
- - - - - -

X - G - X -
- - - - X -
- - - - - -
- - - - - -
X X - P - B
- - - - - -

