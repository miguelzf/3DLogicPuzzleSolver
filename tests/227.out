
Solution:
-  -  -  p  p  p  
-  -  P  p  X  X  
-  C  c  c  c  c  
b  b  b  Y  y  y  
b  X  b  l  L  y  
b  X  b  l  -  X  
================
b  X  b  l  -  X  ||  X  y  G  c  X  p  
b  X  B  l  Y  y  ||  y  y  g  c  X  p  
b  l  l  l  g  g  ||  g  g  g  c  -  p  
b  l  G  g  g  C  ||  c  c  c  c  p  p  
b  l  l  L  X  X  ||  X  X  -  P  p  B  
b  b  b  b  b  b  ||  b  b  b  b  b  b  
